VstsTasks/AzureDtlCreateVM/task-funcs.ps1

```diff
@@ -417,15 +431,35 @@ function Wait-ApplyArtifacts
                 throw "Waited for more than $(ConvertTo-MinutesString $totalWaitMinutes). Failing the task."
             }
 
-            $continueWaiting = $true
-
             $vm = Get-AzureRmResource -ResourceId $ResourceId

```

Comment:
api version?

Reply:
`Api-Version` is not required to get the basic resource information. There are no expansions happening in this call.